### DISCUSSION TITLE
Window is not defined

### DISCUSSION THREAD

**Discussion by AhmedElkhardly** [cuongdevjs/reactjs-social-login#109] [Q&A] [answered]
![image](https://github.com/cuongdevjs/reactjs-social-login/assets/129653196/b457fc4e-c7a9-43b6-aff4-bce3c5a25ab3)
when I try to reload the page it gives me window is undefined in 
node_modules/reactjs-social-login/dist/reactjs-social-login.modern.js:1:410

using:
	"react-social-login-buttons": "^3.9.1",
	"reactjs-social-login": "^2.6.2",
	"next": "12.2.5",

**Comment by cuongdevjs** [comment, marked as answer]
please use it with mode `ssr = false`, you don't need to use ssr for this library. 
Or you can try to check isClient = true before using it.

**Comment by AhmedElkhardly** [comment]
![image](https://github.com/cuongdevjs/reactjs-social-login/assets/129653196/ee8ba45f-ad0b-4ed0-9f0c-f11ffd85300d)
 it work, just by useing `dynamic ` and `ssr = false` and then use `typeof window !== 'undefined' && (Components)`

**Comment by MaoriSL** [comment]
Hi, even by importing with dynamic and setting ssr = false and using typeof window condition, i'm still facing the issue, do you have any solution ?
